sdk/loadtestservice/azure-developer-loadtesting/tests/test_test_runs.py

```diff
@@ -0,0 +1,217 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+import os
+from pathlib import Path
+
+import pytest
+from azure.core.exceptions import ResourceNotFoundError
+
+from testcase import LoadtestingTest, LoadtestingPowerShellPreparer
+from devtools_testutils import recorded_by_proxy, set_custom_default_matcher
+
+test_id = os.environ.get("TEST_ID", "000")
```

Comment:
you shouldn't be using environment variables directly in your test, here is the tests documentation for how to set up and manage your test resources and credentials: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#write-or-run-tests